nasa-gibs/worldview
# Blank map on event projection switch fix

## Description

Fixes #2865 

- Some code cleanup
- Prevent switching to projection that is already active (was causing issue re-rendering event tracks
- Fix issue with map going blank on projection switch while event is selected
- [ ] update test suite

Bonus:
- Add zoomToEvent when proj switches with event selected
- Update deprecated stylelint rule names

## How to test

1. Select "Events" tab
2. Select a sea ice event near arctic or antarctic with multiple dates
3. Select a date other than the default
4. Switch to appropriate polar projection
5. Confirm map shows
6. Confirm map zooms to the selected event
7. Confirm the same date is selected in the sidebar and the corresponding point is centered on the map
